player.py

```diff
@@ -0,0 +1,216 @@
+    
+    def check_user_choice(self, user_choice):
+        '''
+        Checks if the user is entering a valid option, if so, returns the choice, otherwise return None
+        '''
+        try:
+            if user_choice < 0 or user_choice >= len(self.cards):
+                print("Please, enter a valid number")
+                return None
+                        
+            return self.cards[user_choice]
+        
+        except (ValueError, IndexError):
+            return None
+        
+    def get_cards_points(self):
+        """
+        Iterate trough the player's cards and return the total amount of points all the cards
+        values together.
+        """
+        cards_total_points = 0
+        if self.cards == []:
+            return 0
+        
+        for card in self.cards:
+            cards_total_points += card.get_card_points()
+        
+        return cards_total_points
+    
+    def draw_card_to_players_deck(self, current_deck):
+        """
+        Adds a card to the player's card calling the method from the Deck class draw_card()
+        and prints the card the player got.
+        """
+        self.cards.append(current_deck.draw_card())
+        print(f"The card you got is \n {self.cards[-1].name} -- {self.cards[-1].color}")
+    
+    def print_cards(self):
+        """
+        Helper method that prints all the player's card then return the card index resulted
+        from printing all the cards
+        """
+        
+        card_index = 0
+        
+        for card in self.cards:
+            print(f"{card_index} --> {card.name} -- {card.color}")
+            card_index += 1
+            
+        return card_index
+    
+    def remove_card_played(self, card_played):
+        """
+        Removes the card the player played and print Uno! if he/she is with just 2 cards.
+        """
+        card_index = self.cards.index(card_played)
+        if len(self.cards) == 2:
+            print(f"{self.name}: Uno!")
+            
+        del(self.cards[card_index])
+
+import unittest
+from mock import MockInputFunction
+
+class TestPlayer(unittest.TestCase):
+    def setUp(self):
+        self.cards = []
+        self.player = Player("Joao")
+        self.deck = Deck(self.cards)
+        self.special_card_1 = SpecialCard('Wild draw four')
+        self.special_card_2 = SpecialCard('Draw two', 'Yellow')
+        self.special_card_3 = SpecialCard('Reverse', 'Green')
+        self.normal_card_1 = NormalCard('Blue', 'One', 1)
+        self.normal_card_2 = NormalCard('Yellow', 'Six', 6)
+        self.normal_card_3 = NormalCard('Green', 'Zero', 0)
+        self.test_cards = [
+            self.special_card_1,
+            self.special_card_2,
+            self.special_card_3,
+            self.normal_card_1,
+            self.normal_card_2,
+            self.normal_card_3
+        ]
+    
+    def test_add_single_card(self):
+        self.cards.append(NormalCard('Red', 'Test', 5))
```

Comment:
```suggestion 
        test_card = NormalCard('Red', 'Test', 5)
        self.cards.append(test_card)
```
And then you can just assert if self.player.cards[0] is equal test_card